.github/workflows/labeler.yml

```diff
@@ -1,19 +1,13 @@
-name: Pull Request Labeler
-# This workflow is supposed to run every 5 minutes
+# https://github.com/marketplace/actions/labeler
+name: "Pull Request Labeler"
 on:
-  schedule:
-    - cron: '*/20 * * * *'
-permissions:
-  pull-requests:
-    write
+- pull_request_target
+
 jobs:
   triage:
     name: Update PR Labels
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - if: github.repository == 'jupyterlab/jupyterlab'
-        uses: paulfantom/periodic-labeler@dffbc90404f371f76444a69221c2f7ad079e2319
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GITHUB_REPOSITORY: ${{ github.repository }}
-          LABEL_MAPPINGS_FILE: .github/labeler.yml
+    - uses: actions/labeler@main
+      env:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
This is giving "Error: Input required and not supplied: repo-token", for example in https://github.com/jupyterlab/jupyterlab/pull/10311/checks?check_run_id=2714028848:

![image](https://user-images.githubusercontent.com/591645/120274723-8a953900-c2b0-11eb-9d28-c137928df130.png)

Reply:
I forgot to change the with line indeed, because the action used was different.